tests/unit/Affects/Migrations/MigratesHookTest.php

```diff
@@ -14,30 +14,70 @@
 
 namespace Tenancy\Tests\Affects\Migrations;
 
-use Illuminate\Support\Facades\DB;
-use Tenancy\Affects\Migrations\Events\ConfigureMigrations;
-use Tenancy\Affects\Migrations\Providers\ServiceProvider;
 use Tenancy\Facades\Tenancy;
 use Tenancy\Testing\TestCase;
+use Illuminate\Support\Facades\DB;
+use Tenancy\Tenant\Events\Created;
+use Tenancy\Tenant\Events\Deleted;
+use Tenancy\Affects\Migrations\Providers\ServiceProvider;
+use Tenancy\Affects\Migrations\Events\ConfigureMigrations;
+use Tenancy\Database\Drivers\Sqlite\Providers\DatabaseProvider;
 
 class MigratesHookTest extends TestCase
 {
-    protected $additionalProviders = [ServiceProvider::class];
+    protected $additionalProviders = [ServiceProvider::class, DatabaseProvider::class];
```

Comment:
Great example of how easy it is to activate a driver 👍

Reply:
Jup, really loved that!